lighttpd/tox.ini

```diff
@@ -8,7 +8,6 @@ envlist =
 description =
     py{27,37}: e2e ready
 dd_check_style = true
-dd_check_style_logging_format = false
```

Comment:
Note: `dd_check_style_logging_format` defaults to true, so removing this line will enable lint for logging format.